### PR TITLE
Brand guidelines

### PR DESCRIPTION
**Pull request by estensen**
Todo
- [x] Add brand guidelines
- [x] Add zip with highres logos
- [x] Add examples of how the brand shouldn't be used (see https://slack.com/brand-guidelines)

### CODE REVIEW

File: app/routes/brand/components/BrandPage.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .root {
-  color: red;
+  color: #777;
```
Comment (SmithPeder) [inline app/routes/brand/components/BrandPage.css]: Preferable use global css vars from `app/styles/variables.css` and add new one as needed.

File: app/routes/brand/components/BrandPage.js [hunk 2]
```diff
@@ -1,19 +1,76 @@
-import React, { Component } from 'react';
+import React from 'react';
+
 import { Content } from 'app/components/Content';
 import NavigationTab from 'app/components/NavigationTab';
 import styles from './BrandPage.css';
+import Button from '../../../components/Button';
```
Comment (SmithPeder) [inline app/routes/brand/components/BrandPage.js]: `app/components/Button`

**Comment by estensen** [comment]
Probably gonna add some whitespace over the images, but feel free to give me feedback on how it looks. I'm no expert in Flexbox either.

screenshot 2018-11-10 14 43 41

**Comment by SmithPeder** [comment]
Looks good 💯 
Have you thought about adding some references to the composition off ball and text? Can one move the ball? Can one increase the size of the ball? Can one like make the ball real big and put the text on the ball? Can one use the ball by itself without the text?

**Comment by estensen** [comment]
Feel free to merge if you're satisfied

screenshot 2018-11-14 23 40 06

**Pull request merged**